Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ColliderType.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization
+{
+    public enum ColliderType
+    {
+        None,
```

Comment:
please add summary tags

Reply:
Wasn't able to find this on khronos declaration.